src/test/java/com/hermesworld/ais/galapagos/topics/controller/TopicControllerTest.java

```diff
@@ -55,6 +55,8 @@ class TopicControllerTest {
 
     private TopicBasedRepositoryMock<TopicMetadata> topicRepository;
 
+    private Boolean schemaDeleteWithSub;
```

Comment:
Are you always putting in `null` for this value in test cases? Completely missing test cases for the new method, testing all relevant combinations.